ConsoleApp/Stack.cs

```diff
@@ -0,0 +1,46 @@
+namespace ConsoleApp;
+
+public class Stack<T>
+{
+    record Item
+    {
+        public T Value { get; set; }
+        public Item Next { get; set; }
+    }
+    private Item _top;
+    public int Length { get; set; }
```

Comment:
instead, it should be property with the name Count
```suggestion
    public int Count { get; set; }
```